release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -3301,4 +3301,23 @@ $ oc adm release info 4.12.6 --pullspecs
 [id="ocp-4-12-6-updating"]
 ==== Updating
 
+To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-12-7"]
+=== RHBA-2023:1163 - {product-title} 4.12.7 bug fix and security update
+
+Issued: 2023-03-13
+
+{product-title} release 4.12.7, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:1163[RHBA-2023:1163] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:1162[RHBA-2023:1162] advisory.
```

Comment:
is this a security update? It looks like both RPM and images are RHBA

Reply:
Thanks for the review, @opayne1 . Well spotted. I updated the entry title. I think not make it plural, because you mentioned not to do this on another z-stream I worked on.